src/pages/GamePage.js

```diff
@@ -204,6 +207,8 @@ function GamePage({ match }) {
           if (vals.length === 3) {
             if (checkSet(...vals)) {
               handleSet(vals);
+              //Success State
```

Comment:
What does this comment mean?

Reply:
I put it to remind myself where can I play the sound but forgot to remove it :D
Just removed it and followed the importing pattern.

Reply:
Idk how to cite the source.
I used https://notificationsounds.com/about

Reply:
@ekzhang 
I cant fix this conflict.

Reply:
Available under a Creative Commons Attribution license, so we will need to provide attribution back to the site, for example in `AboutPage`.